source/new-starter/install-tools/index.html.md.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Install developer tools
+weight: 23
+last_reviewed_on: 2020-12-10
+review_in: 6 months
+---
+
+# Install developer tools
+
+The GOV.UK Account team uses the following developer tools:
+
+- `gds-cli`, the Government Digital Service Command Line Interface (CLI)](https://github.com/alphagov/gds-cli) to connect to the VPN and update secrets
+- fly, a CLI to interact with Concourse
+- the CloudFoundry CLI to interact with our servers on the GOV.UK PaaS
+
+When you join the GOV.UK Account team, you must install these tools.
+
+## Install the gds-cli
+
+The `gds-cli` is a CLI to connect to the VPN and update secrets.
```

Comment:
Can we link to (or add docs) on how to connect to the VPN and update secrets using `gds-cli`?  I'm not sure we've covered this, and a new starter probably won't know how to do this.

Reply:
Will add links to new team manual docs on rotating secrets and gov.uk developer docs on connecting to the VPn